java/src/jmri/jmrit/symbolicprog/tabbedframe/PaneProgPane.java

```diff
@@ -2418,6 +2418,9 @@ public void newVariable(Element var, JComponent col,
 
         // create the paired label
         JLabel l = new WatchingLabel(" " + label + " ", rep);
```

Comment:
> In java/src/jmri/jmrit/symbolicprog/tabbedframe/PaneProgPane.java:
> 
> > @@ -2418,6 +2418,9 @@ public void newVariable(Element var, JComponent col,
> >  
> >          // create the paired label
> >          JLabel l = new WatchingLabel(" " + label + " ", rep);
> Just out of curiosity, why are we placing spaces around the label at all?
> 
I suspect it was a (possibly misguided) attempt to create extra white space between adjacent labels/variables.

My first inclination was to get rid of it, but decided to take the more conservative approach rather than run the risk of breaking existing pane layouts.

I was under domestic pressure to get away from the keyboard and I needed a quick fix as some of my subsequent XML refactoring depended on it. I will at least tidy it up by moving the literal into a static constant (as I later remembered I had done with tool tips in the addTextHTMLaware method).

I have written three public static string utility methods in PaneProgPane, at least two of which I use in other classes and I have been planning to move them elsewhere, possibly to the XMLUtil class.